src/main/resources/api.yml

```diff
@@ -0,0 +1,503 @@
+info:
+  title: LostInBerlin API
+  description: Move your app forward with the LostInBerlin API
+  version: "1.0.0"
+# the domain of the service
+host: lost-in-berlin.com
+# array of all schemes that your API supports
+schemes:
+- https
+# will be prefixed to all paths
+basePath: /api/v1
+securityDefinitions:
+  apikey:
+    type: token
+    name: server_token
+    in: header
+produces:
+- application/json
+paths:
+  /business:
+    get:
+      summary: Existing business profiles
+      description: The endpoint returns information about business profiles available at a given location.
+      The response includes the name and other details about each business, and lists the profiles
+      in the proper display order.
+      parameters:
+      - name: location
+        in: query
+        description: Given location.
+        required: true
+        type: string
+      security:
+      - token: []
+      tags:
+      - business
+      responses:
+        "200":
+          description: An array of profiles
+          schema:
+            type: array
+            profiles:
+              $ref: '#/definitions/Business'
```

Comment:
I think here should be term 'items' instead of 'profiles'. 'items' is a reserved word for denoting elements of the array.

Reply:
Done